redisgears_core/src/lib.rs

```diff
@@ -590,7 +589,13 @@ impl DbPolicy {
     }
 }
 
+struct RedisVersion {
```

Comment:
Can we add `#[derive(Debug, Copy, Clone)]` on top?

Reply:
Copy we can not because Version do not derive Copy. But we sure can Clone and Debug.

Reply:
Actually, Version also do not implement Clone.